packages/react/src/components/Header/HeaderActionGroup.jsx

```diff
@@ -16,19 +24,146 @@ const propTypes = {
  * or dropdown menus, passing an onToggleExpansion to each action
  */
 const HeaderActionGroup = ({ actionItems }) => {
+  const overFlowContainerRef = useRef(null);
+  const [overflowItems, setOverflowItems] = useState([]);
+  const breakpoint = useRef(null);
+  const [overflowOpen, setOverflowOpen] = useState(false);
+  const [showMenu, setShowMenu] = useState(false);
+  const [menu, setMenu] = useState(null);
+
+  const checkForOverflow = useCallback(() => {
+    /* istanbul ignore else */
+    if (overFlowContainerRef.current) {
+      // the first button in the action group
```

Comment:
Can we skip the comment and change the code to say the same thing?

Reply:
Done.